src/containers/subjects-launch/SubjectGPLLaunch/index.tsx

```diff
@@ -178,6 +159,14 @@ async function checkGPLTriggerAllow(gdsResults: GDSRow[]): Promise<GplLaunchChec
     return gplCheck;
   }
 
+  // Check if Subject has FPPE
+  const filterFPPE = limsResults.filter((r) => r.source == 'FFPE');
+  if (Array.isArray(filterFPPE) && filterFPPE.length) {
+    gplCheck.isGplLaunchAllowed = false;
+    gplCheck.message = `Subject has FPPE source found. GPL doesn't with FPPE sample.`;
+    return gplCheck;
+  }
+
```

Comment:
Typo
```
Subject has FPPE source found. GPL doesn't work with FPPE tumor sample.
```

Reply:
Also. Could you improve check logic a bit, such that -- I think, we can't just simply rely on FFPE tumor library found on a Subject. A Subject could still have FFPE and, other sources `blood` or `tissue`, etc. In that case, we might still need to allow GPL trigger. @scwatts could you help confirm it, please..?

An example case, try `SBJ00699`.

https://portal.umccr.org/subjects/SBJ00699/overview

Reply:
Since the `gpl_submit_job` Lambda will exit early when multiple tumor samples are detected, there's no need to check sample type for individual tumor samples. That Lambda was meant to provide some automation but was limited without the ability to interact with or receive additional input from the user; I had intended for the `submit_job_manual` to eventually be connected to the data portal and have sample selection etc moved to the portal UI. We'll be deprecating the GPL pipeline soon, so not much value doing that at this point.

Reply:
Okay.! In that case, Will; after fixing typo then, can we keep this check routine commented out for now..? i.e. let it continue trigger from UI. We can show proper message such that the trigger will be failed to user. But, let that be as-is that way for now...

Reply:
Or, just pull out warning message there, saying if FFPE, the triggered job may eventually be failed or not succeeded.

https://primereact.org/message/

Reply:
I'm not sure of the other changes being made along with these here but does it make sense to set the eligibility checks as (1) single tumor sample and (2) tumor sample type not FFPE?

Reply:
Think we can also check that way too.. Right, Will..?

Reply:
Yes, we could!